torchao/csrc/metal/mps/OperationUtils.mm

```diff
@@ -0,0 +1,14 @@
+#include "OperationUtils.h"
+
+namespace torchao::mps {
+  id<MTLBuffer> getMTLBufferStorage(const uint8_t * data, size_t nbytes) {
+    return [MPSDevice::getInstance()->device() newBufferWithBytesNoCopy:(void*)data
+                                                                length:nbytes
+                                                                options:0
```

Comment:
From function signature, it looks like you are trying to allocate a read-only buffer, but at the same time options are null.

Also, from an API perspective, passing a raw pointer with no liftetime guarantees feels wrong. Can it be converted to say `shared_ptr` and deallocator will dec-ref it?

Reply:
I think I'll ended up just calling PyTorch's `mtl_setBuffer()`

Reply:
Is the code in OperationUtils.mm specific to pytorch?